flurry-client/navigation/DriverTabNavigator.js

```diff
@@ -25,13 +26,13 @@ export default function MainTabNavigator() {
         }}
       />
       <Tab.Screen
-        name="Analytics"
-        component={AnalyticsScreen}
+        name="Self-Report"
+        component={SelfReportScreen}
         options={{
           tabBarIcon: ({ focused }) => (
-            <Ionicons
+            <MaterialIcons
               color={focused ? "blue" : "gray"}
```

Comment:
nit: could you change this from "blue" to the "themedBlue" that exists in the styles file?